zerver/models.py

```diff
@@ -953,6 +953,16 @@ class RealmFilter(models.Model):
     realm: Realm = models.ForeignKey(Realm, on_delete=CASCADE)
     pattern: str = models.TextField()
     url_format_string: str = models.TextField(validators=[URLValidator(), filter_format_validator])
+    render_format_string: str = models.TextField(
+        default="",
```

Comment:
I chose the `default=""` and not `null=True` as this may create some issue when the linkifier data will be used for comparison purpose (@ frontend), or in other cases, this may cause some unexpected problems like errors are thrown (@ frontend).

Reply:
Can you explain the reasoning more, perhaps with an example problem you'd expect with `null=True`?

Reply:
We probably need to put some unnecessary check at (when `null=True` is used):
1. https://github.com/zulip/zulip/blob/4cff56a47c65d8f2a366cc948ebd2e2408c9a802/static/js/settings_linkifiers.js#L103-L104.
2. https://github.com/zulip/zulip/blob/4cff56a47c65d8f2a366cc948ebd2e2408c9a802/static/js/settings_linkifiers.js#L30.

Hence I found that empty strings would be much better rather than providing unnecessary checks.